cpp/CMakeLists.txt

```diff
@@ -88,10 +88,20 @@ else()
   set(ARROW_FULL_SO_VERSION "${ARROW_SO_VERSION}.${ARROW_VERSION_PATCH}.0")
 endif()
 
+set(ARROW_FORMAT_VERSION "1.0.0")
+# Get the major and minor version numbers
+string(REGEX REPLACE "\\." ";" _ARROW_FORMAT_VERSION_LIST ${ARROW_FORMAT_VERSION})
+list(GET _ARROW_FORMAT_VERSION_LIST 0 ARROW_FORMAT_VERSION_MAJOR)
+list(GET _ARROW_FORMAT_VERSION_LIST 1 ARROW_FORMAT_VERSION_MINOR)
+list(GET _ARROW_FORMAT_VERSION_LIST 2 ARROW_FORMAT_VERSION_PATCH)
+
 message(STATUS "Arrow version: "
                "${ARROW_VERSION_MAJOR}.${ARROW_VERSION_MINOR}.${ARROW_VERSION_PATCH} "
                "(full: '${ARROW_VERSION}')")
 message(STATUS "Arrow SO version: ${ARROW_SO_VERSION} (full: ${ARROW_FULL_SO_VERSION})")
+message(STATUS "Arrow Format version: ${ARROW_FORMAT_VERSION_MAJOR}."
+               "${ARROW_FORMAT_VERSION_MINOR}.${ARROW_FORMAT_VERSION_PATCH} "
+               "(full: '${ARROW_FORMAT_VERSION}')")
```

Comment:
We can just show `${ARROW_FORMAT_VERSION}` here. Because `${ARROW_FORMAT_VERSION}` never have `-SNAPSHOT`.